features/bookmarks/bookmark_create.feature

```diff
@@ -48,7 +50,99 @@ Scenario: Create a bookmark
     Then I should not see "I liked this story"
     When I go to first_bookmark_user's user page
     Then I should not see "I liked this story"
-      
+
+Scenario: Create a series bookmark
+  Given I am logged in as "first_bookmark_user"
+  When I am on first_bookmark_user's user page
+  Then I should see "have anything posted under this name yet"
+  When I am logged in as "another_bookmark_user"
+    And I post the work "Revenge of the Sith"
+    And I add the work "Revenge of the sith" to the series "Star Wars"
+    And I go to the bookmarks page
+  Then I should not see "Star Wars"
+  When I am logged in as "first_bookmark_user"
+    And I go to the series page
+    And I follow "Star Wars"
+  Then I should see "Bookmark Series"
+  When I follow "Bookmark Series"
+    And I fill in "bookmark_notes" with "I liked this trilogy"
+    And I fill in "bookmark_tag_string" with "This is a series tag, and another series tag,"
+    And I check "bookmark_rec"
+    And I press "Create"
+  Then I should see "Bookmark was successfully created"
+    And I should see "My Bookmarks"
+  When I am logged in as "another_bookmark_user"
+    And I go to the bookmarks page
+    And "AO3-5221" is fixed
```

Comment:
It turns out this isn't due to AO3-5221. It's because the index uses `Bookmark.latest` if the caching admin setting isn't enabled, and the `latest` scope only grabs works.